Dungeoneer/src/com/interrupt/dungeoneer/screens/MainMenuScreen.java

```diff
@@ -593,15 +584,24 @@ public void clicked(InputEvent event, float x, float y) {
 		progress[selectedSave] = null;
 		deleteSavegame(selectedSave);
 		selectedSave = null;
-	}
+    }
+    
+    /** Handles the event when the options button is clicked. */
+    private void handleOptionsButtonEvent() {
+        GameApplication.SetScreen(new OverlayWrapperScreen(new OptionsOverlay(false, true)));
+    }
+
+    /** Handles the event when the mods button is clicked. */
+    private void handleModsButtonEvent() {
+        GameApplication.SetScreen(new OverlayWrapperScreen(new ModsOverlay()));
+    }
 	
 	private void loadSavegames() {
-		String baseSaveDir = "save/";
-		FileHandle dir = Game.getFile(baseSaveDir);
+		FileHandle dir = Game.getFile(BASE_SAVE_DIR);
 		
-		Gdx.app.log("DelverLifeCycle", "Getting savegames from " + dir.path());
+		Gdx.app.log(LOG_TAG_DELVER_LIFE_CYCLE, "Getting savegames from " + dir.path());
```

Comment:
I'd prefer to keep the string literal here, unless you feel strongly about it?

Reply:
Oh yeah. That was past-me trying to silence SonarLint. The standard of having the string literals directly where they are needed looks definitely better.

Reply:
Fixed. Thanks!